### PR TITLE
ModalInvoke: Customization of button labels is added (#633)

### PR DESCRIPTION
**Pull request by DRITE**
see #633

### CODE REVIEW

File: src/components/ModalInvoke/ModalInvoke.tsx [hunk 1]
```diff
@@ -71,12 +71,16 @@ const ModalInvoke: React.FunctionComponent<ModalInvokeProps> = props => {
         }
     }
 
+    const okLabel = props.confirmOperation.okText || t('Ok')
```
Comment (Dergash) [inline src/components/ModalInvoke/ModalInvoke.tsx:74]: Optional chaining probably required for `confirmOperation` (based on original line 78)

File: src/interfaces/operation.ts [hunk 2]
```diff
@@ -205,20 +205,7 @@ export enum OperationPostInvokeConfirmType {
...
-    /**
-     * Title for modal
-     */
-    message: string
-    /**
-     * Additional text for modal
-     */
-    messageContent?: string
-}
+export type OperationPostInvokeConfirm = OperationModalInvokeConfirm
```
Comment (Dergash) [inline src/interfaces/operation.ts:208]: I think we should reverse it: the key part is that we've received post invoke for confirmation, and then we can decide how to handle it. By default we handle it with modal window, but client application might be able to handle this type of post actions via popups, notifications or any other way.

So I think we should keep `OperationPostInvokeConfirm` and deprecate through alias `OperationModalInvokeConfirm`.

File: src/interfaces/operation.ts [hunk 3]
```diff
@@ -229,13 +216,21 @@ export interface OperationModalInvokeConfirm {
      */
     type: OperationPostInvokeConfirmType | OperationPreInvokeType | string
     /**
-     * Title for modal
+     * Body text of a modal
      */
     message: string
     /**
-     * Additional text for modal
+     * Custom modal title
```
Comment (Dergash) [inline src/interfaces/operation.ts:223]: Well that's misleading all right. Probably TODO for renaming will be appropriate.

**Review by Dergash — CHANGES_REQUESTED** [review]

File: src/components/ModalInvoke/ModalInvoke.tsx (hunk 1 above)
Comment (DRITE) [reply src/components/ModalInvoke/ModalInvoke.tsx:74]: Optional chaining is added

File: src/interfaces/operation.ts (hunk 2 above)
Comment (DRITE) [reply src/interfaces/operation.ts:208]: resolved

File: src/interfaces/operation.ts (hunk 3 above)
Comment (DRITE) [reply src/interfaces/operation.ts:223]: todo is added

**Review by Dergash — APPROVED** [review]

**Pull request merged**